workshop/content/30-python/30-hello-cdk/200-lambda.md

```diff
@@ -198,7 +198,7 @@ Let's go to the AWS Lambda Console and test our function.
 
     ![](./lambda-2.png)
 
-4. Select __Amazon API Gateway AWS Proxy__ from the __Event template__ list.
+4. Select __apigateway-aws-proxy__ from the __Event template__ list.
```

Comment:
The template's actual name is `apigateway-aws-proxy`, but the dropdown the user will see still shows the full name `API Gateway AWS Proxy`
<img width="297" alt="Screen Shot 2022-06-17 at 4 39 27 PM" src="https://user-images.githubusercontent.com/44349620/174412880-88090198-5814-4330-ab9f-85836223a0f7.png">

I think I'd prefer to either clarify both, or keep the name as the same as the name seen in the dropdown menu